notebooks/official/ml_metadata/vertex-pipelines-ml-metadata.ipynb

```diff
@@ -895,7 +889,7 @@
         "id": "cc15017be48e"
       },
       "source": [
-        "After running this cell, you'll see a link to view each pipeline in the Google Cloud console. Open that link to see more details on your pipeline.\n",
+        "After running this cell, you can see a link to view each pipeline in the Google Cloud console. Open that link to get more details about your pipeline.\n",
```

Comment:
- change: "After running this cell, you can see a link to view each pipeline in the Google Cloud console"
    - to: "After running this cell, there is a link to view each pipeline in the Google Cloud console"